packages/components/src/internal/components/buttons/ResponsiveMenuButtonGroup.tsx

```diff
@@ -9,12 +9,14 @@ interface ResponsiveMenuItem {
 }
 
 interface Props {
+    asSubMenu?: boolean;
     items: ResponsiveMenuItem[];
+    subMenuWidth?: number;
     user: User;
 }
 
 export const ResponsiveMenuButtonGroup: FC<Props> = memo(props => {
-    const { items, user } = props;
+    const { items, user, subMenuWidth = 1600, asSubMenu = true } = props;
```

Comment:
Is there an algorithm to determine the appropriate subMenuWidth? I see some width are reused, such as 700, 1300.. Can those be shared enums with description/comments?

Reply:
No algorithm. The values were set on a case by case basis as it all depends on the context of what other grid buttons are present (sometimes it has a delete/remove, sometimes it has an add, etc.).